### DISCUSSION TITLE
Unable to deploy into nillion testnet

### DISCUSSION THREAD

**Discussion by ghoshvidip26** [NillionNetwork/discussions#124] [General]
<img width="1253" alt="Screenshot 2024-10-05 at 17 44 09" src="https://github.com/user-attachments/assets/84151f3b-8765-4dcc-8655-5cf3632ca0dd">
While I am trying for the vercel deployment, I am getting this error. Dist file error.

**Comment by wwwehr** [comment]
Hi @Vidip-Ghosh !

It appears you're having trouble deploying to vercel which is a hosting platform for websites. Can you describe exactly which workflow you are working on? Is your repo open source? 

[here](https://nextjs.org/docs/pages/building-your-application/deploying) is a good resource for figuring out vercel setup.
